Gist/Managers/MessageQueueManager.swift

```diff
@@ -3,6 +3,7 @@ import UIKit
 
 class MessageQueueManager {
     private var queueTimer: Timer!
+    private var localMessageStore: [String: Message] = [:]
```

Comment:
I think this code would benefit from a comment explaining what value this store provides. 

Explaining why this feature got added would be helpful for future reference.

Reply:
Added a comment explaining what the local store is used for